lib/rules/base.js

```diff
@@ -69,6 +75,32 @@ module.exports = class Base {
     return this._configResolver.editorConfig();
   }
 
+  get _moduleId() {
+    const moduleIdKey = `${this.constructor.name}:moduleId`;
+
+    if (!loggedModules.has(moduleIdKey)) {
+      this._console.log('The `_moduleId` property is deprecated. Please use `_filePath` instead.');
+
+      this.loggedModules.push(moduleIdKey);
+    }
+
+    return this[MODULE_ID];
+  }
+
+  get _moduleName() {
+    const moduleNameKey = `${this.constructor.name}:moduleName`;
+
+    if (!loggedModules.has(moduleNameKey)) {
```

Comment:
We want to change this around to make it only log once _per rule_ (not once per module being linted, which is what is implemented here I think).

Off the top of my head, that would be something like:

```js

if (!loggedRules.has(this.ruleName)) {
  this._console.log(
    `The \`_moduleName\` property is deprecated (accessed from ${this.ruleName}). Please use \`_filePath\` instead.`
  );

  loggedRules.add(this.ruleName);
}
```

Reply:
This is one log per Rule class, not once per module linted.

`ruleName` would work too, but it's effectively the same thing.

Reply:
Changed to `ruleName` 🤷‍♂️

Reply:
thanks sorry, I originally misread the diff